auth/auth_stub.go

```diff
@@ -22,7 +22,12 @@ package auth
 
 import "context"
 
-var _ Client = &noop{}
+var (
+	// NoopClient is used for testing and no-op integration
+	NoopClient = noopClient(nil)
```

Comment:
+1 This has been bothering me as well. We need to decide on a convention for uberfx atleast.